front/src/common/Map/const.ts

```diff
@@ -105,5 +105,49 @@ export const SEARCH_URL = `/editoast/search`;
 
 export type searchPayloadType = {
   object: string;
-  query: (string | any[])[];
+  query: (string | boolean | any[])[];
```

Comment:
```suggestion
  query: SearchQuery;
```
Can you use this type (from osrdEditoastApi) ? It would be more consistent !

Reply:
Done